tests/test_clos/test_project_id.py

```diff
@@ -0,0 +1,57 @@
+"""Pytest added for checking project_id"""
```

Comment:
```suggestion
"""Pytest added for checking Job.project_id"""
```
Not really required as the filename is quite self-explanatory.